### DISCUSSION TITLE
Software stack

### DISCUSSION THREAD

**Discussion by samgriek** [daveshap/ACE_Framework#31] [General]
What will the ACE Framework provide in terms of infrastructure implementation?  Will there be a local dev environment that someone can setup on their PC to begin development?  If so, what tech stack decisions are already made?

It would be ideal if it was built on Open Source technology as much as possible IMO.

It would be great if a light containerization technology was used (assuming models are served remotely).

Is the goal  to provide the infra as open-source or for implementations to roll their own?

I've already built my local agent infra based on the white paper, but I don't want to diverge too much if you are going to provide the infrastructure...

**Comment by daveshap** [comment]
The ACE Framework is a blueprint, so don't worry. Build however you like. The idea is that there are plenty of ways to implement it.
